src/utils/communitySbt/getScores.ts

```diff
@@ -0,0 +1,47 @@
+import { getSeasonUsers } from '@voltz-protocol/subgraph-data';
+
+export type GetScoresArgs = {
+  season: number;
+  subgraphUrl: string;
+  ignoredWalletIds: Record<string, boolean>;
+};
+
+export type ScoreResult = {
+  lpScores: Record<string, number>;
+  traderScores: Record<string, number>;
+};
+
+/**
+ * @dev Query the Main subgraph and retrieve season's liquidity
+ * or trading score of all users based on time weighted liquidity.
+ * Score is based on swaps for Traders and mints/burns for LPs.
+ */
+export async function getScores({
```

Comment:
for leaderboard, we are only interested in trader scores. do you think also computing LP scores can affect performance?

Reply:
That's right, but all data is fetched at once so the difference is quite insignificant.